src/mixins/date/index.js

```diff
@@ -0,0 +1,5 @@
+import moment from 'moment'
+
+export const firebaseTimestampToDate = (timestamp) => (
+  moment(timestamp.seconds * 1000)
+)
```

Comment:
This returns a `Moment` object, not a `Date` object. Consider making that clear on the function name.